### DISCUSSION TITLE
Sync History

### DISCUSSION THREAD

**Discussion by mrburns-42** [LeoKlaus/plappa#162] [Feature Requests]
Currently, the playback history is only saved locally in Plappa. However, if I now change the device, I would like to access it. I would like it if it could be synced between devices, for example via iCloud.

It might make sense for this option to be optional after a possible implementation, at least if this is to run via the iCloud. For example, that you can switch it on or off via the app settings.

**Comment by LeoKlaus** [comment]
I've implemented this, it should come with the next TestFlight release.

**Reply by mrburns-42** [reply]
Thanks!
